answerking_app/models/serializers.py

```diff
@@ -0,0 +1,157 @@
+import re
+
+from django.core.validators import (
+    MaxValueValidator,
+    MinValueValidator,
+    RegexValidator,
+)
+from rest_framework import serializers
+
+from answerking_app.models.models import Category, Item, Order, OrderLine
+
+MAXNUMBERSIZE = 2147483647
+
+
+def compress_white_spaces(value: str) -> str:
+    return re.sub(" +", " ", value)
+
+
+class ItemSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(
+        max_length=50,
+        allow_blank=False,
+        validators=[RegexValidator("^[a-zA-Z !]+$")],
+        trim_whitespace=True,
+    )
+    price = serializers.DecimalField(
+        max_digits=19,
+        decimal_places=2,
+        validators=[MinValueValidator(0), MaxValueValidator(MAXNUMBERSIZE)],
+    )
+    description = serializers.CharField(
+        max_length=200,
```

Comment:
need to change this if we change model field

Reply:
not sure what you mean

Reply:
see comment above